Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/colmartech_bundles.yml

```diff
@@ -113,3 +113,27 @@
     - CMVendorHeadsetSquad
     - CMHandsBlackMarine
     - CMBootsBlackFilled
+
+- type: entity
+  parent: CMVendorBundleRiflemanApparel
+  id: RMCVendorBundleMilitaryPoliceApparel
+  name: standard military police apparel
+  components:
+  - type: CMVendorBundle
+    bundle:
+    - CMHandsBlackMarine
+    - CMHeadsetMPO
+    - CMBootsBlackFilled
+    - CMHeadBeretRed
```

Comment:
this should be MP helmet, and some of these items are not in the bundle, armor is in a different section
![image](https://github.com/RMC-14/RMC-14/assets/121047731/d0f56c1c-f99c-4151-a483-d8ddf3bb32a2)

Reply:
oops, done.